projects/github-actions/pr-is-up-to-date/run.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eo pipefail
 
-source ./funcs.sh
+source "$BASE/funcs.sh"
```

Comment:
You need to set `$BASE` somewhere.
```suggestion
BASE=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
source "$BASE/funcs.sh"
```

Reply:
That's...weird that I did but for some reason it wasn't committed.